idacredential/src/commonMain/kotlin/at/asitplus/wallet/idaustria/BpkIntermediateValues.kt

```diff
@@ -0,0 +1,53 @@
+package at.asitplus.wallet.idaustria
+
+import at.asitplus.wallet.lib.jws.ByteArrayBase64UrlSerializer
+import kotlinx.serialization.KSerializer
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+import kotlinx.serialization.descriptors.PrimitiveKind
+import kotlinx.serialization.descriptors.PrimitiveSerialDescriptor
+import kotlinx.serialization.descriptors.SerialDescriptor
+import kotlinx.serialization.encodeToString
+import kotlinx.serialization.encoding.Decoder
+import kotlinx.serialization.encoding.Encoder
+import kotlinx.serialization.json.Json
+
+object BpkIntermediateValueSerializer : KSerializer<BpkIntermediateValues> {
+    override val descriptor: SerialDescriptor =
+        PrimitiveSerialDescriptor("BpkIntermediateValueSerializer", PrimitiveKind.STRING)
+
+    override fun serialize(encoder: Encoder, value: BpkIntermediateValues) =
+        encoder.encodeString(Json {}.encodeToString(value))
+
+    override fun deserialize(decoder: Decoder): BpkIntermediateValues =
+        Json{}.decodeFromString(decoder.decodeString())
+}
+
+@Serializable
```

Comment:
Is there a way i can add `@Serializable(with = BpkIntermediateValueSerializer::class)` here without at the same time causing an infinite loop in the serializer class (since `encodeToString` would then also use the with= annotation)?

Essentially, what I'm trying to do is force `.encodeToString` to use the "default" JSON serializer instead of the custom. 

Any ideas @JesusMcCloud?